sleap/gui/commands.py

```diff
@@ -2989,15 +2989,28 @@ def set_visible_nodes(
 
         has_missing_nodes = False
 
+        #calculate scale factor for getting new x and y values
```

Comment:
[Google style comments](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) por favor